src/test/scala/com/reagroup/appliedscala/urls/repositories/Http4sMetascoreRepositorySpec.scala

```diff
@@ -0,0 +1,48 @@
+package com.reagroup.appliedscala.urls.repositories
+
+import org.specs2.mutable.Specification
+import cats.data.IdT
```

Comment:
What is this? Do we need it?

Reply:
Nope. I was trying to use `Id` instead of `IO` before. Will remove.